unitary/examples/quantum_chinese_chess/move_test.py

```diff
@@ -261,3 +261,113 @@ def test_jump_basic():
     assert len(board_probabilities) == 2
     assert_fifty_fifty(board_probabilities, locations_to_bitboard(["d1"]))
     assert_fifty_fifty(board_probabilities, locations_to_bitboard(["a3"]))
+
+
+def test_split_jump_classical_source():
```

Comment:
Do you think it is worth testing the is_entangled / reset functionality to make sure it is correct?

Reply:
Thanks Doug. It's added in the test.